README.md

```diff
@@ -54,6 +54,9 @@ Examples of template locations:
   one of the inputs declared by the template in its `spec.yaml`. May be repeated
   to provide multiple inputs, like
   `--input=name=alice --input=email=alice@example.com`.
+- `--input-file=file`: provide a yaml file with input(s) to the template. If `key`
```

Comment:
Could we mention that this file should contain a map where all the values must be strings?

Reply:
Done.